src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java

```diff
@@ -499,6 +505,12 @@ private static Player newPlayerWithName(String name) {
         return player;
     }
 
+    private static Player newPlayerWithAddress(String ip) {
```

Comment:
Please use the method in TestUtils instead

Reply:
can't find that method :/

Reply:
TestHelper#mockPlayerIp

Reply:
Ok fixing that, test are failing due to somereflection issues with BukkitTasks :/ could you take a look ? Probably something changed in bukkit.

Reply:
Not atm, I'm abroad for the upcoming three weeks

Reply:
Not atm, I'm abroad for the next three weeks

Reply:
Ok, i'll try to fix them